### PR TITLE
issue/1013

### PR DESCRIPTION
**Pull request by savger1024**
There is recipes route, There is recipes:id route, the database connection probably works, Dependency installations, dotenv, express, mysql, nodemon

### CODE REVIEW

File: Backend/app.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+});
+connection.connect(function (err) {
+  if (err) throw err;
+  console.log("Connected!");
+});
+
+app.get("/", function (req, res) {
+  res.send(process.env.NAMEOFSQL || "NodeJS runs");
+});
+app.get("/recipies", function (req, res) {
```
Comment (Axi20) [inline Backend/app.js]: Please change this to "/recipes"

File: Backend/app.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  connection.query("SELECT * FROM recipes", function (error, results, fields) {
+    if (error) throw error;
+    console.log("The result is: ", results);
+  });
+  const userData = users.find((u) => u.id === 1);
+  res.send(userData);
+});
+app.get("/recipies/:id", function (req, res) {
```
Comment (Axi20) [inline Backend/app.js]: Please change this to "/recipes:id"

File: Backend/app.js (hunk 1 above)
Comment (savger1024) [reply Backend/app.js]: Done

File: Backend/app.js (hunk 2 above)
Comment (savger1024) [reply Backend/app.js]: Done

**Review by Axi20 — APPROVED** [review]
Backend endpoints created and approved.

**Pull request merged**